.gitpod.yml

```diff
@@ -0,0 +1,9 @@
+image:
+  file: .docker/gitpod/mathlib/Dockerfile
+
+vscode:
+  extensions:
+    - jroesch.lean@0.16.24:wYErjZXBf8vkagq/BhUHJA==
```

Comment:
Do we need to update this every time the vscode-lean extension is updated?  Do old PRs then only have access to outdated versions of the extension?

Reply:
It seems so, but I haven't tested what happens if you leave off the `@...`.

My instinct is that typically PRs move much faster than the extension, so this isn't catastrophic. Moreover, you can just update the extension inside the session if you really want to.

Reply:
To be fair, I'm mostly concerned about the additional workload of updating the extension version here.  Ideally we could get rid of the `@...` here.

Reply:
It seems to work fine without the `@...`, so I'm going to do that and hope for the best.